corehq/apps/user_importer/importer.py

```diff
@@ -653,10 +655,14 @@ def _process_web_user(self):
                     web_user_importer.add_to_domain(role_qualified_id, self.user.location_id,
                                                 self.user.assigned_location_ids)
                 elif not web_user or not web_user.is_member_of(self.domain):
+                    profile = None
+                    if cv["profile_name"] in self.domain_info.profiles_by_name:
```

Comment:
Should this report an error/warning if the profile isn't found?

Reply:
that would be a nice feedback to have [f1b723c](https://github.com/dimagi/commcare-hq/pull/34466/commits/f1b723c73d2878e52a47d4f6fe8cf141abd8c5ca)

Reply:
beautiful